source/style-guide/release-notes-guidelines.txt

```diff
@@ -158,36 +158,34 @@ Use the following guideline when creating items in each section:
      - Use sentences to describe the new feature or enhancement.
        Provide as needed, and provide a link to any section in the
        documentation that describes that feature.
-     - IPv6 support has been added. When you create a server, an IPv4
-       address and an IPv6 address block are assigned. The IPv4 address and
-       IPv6 address block are used for the public interface (Internet). The
-       IPv4 address is also used for the private interface.
+     - Supports IPv6. When you create a server, you can specify 
+       both IPv4 and IPv6 addresses.
 
    * - Resolved issues
      - Provide an initial phrase that describes the issue that was
        fixed. Start the phrase with a past-tense verb. If necessary,
        include sentences to further explain the fix. If you list only
        phrases, do not use ending punctuation.
-     - Added support for Debian 8 (Jessie)
+     - - Supports Debian 8 (Jessie)
 
-       Trimmed the Monitoring ID and Monitoring Token configuration
-       variables to ensure correctness
+       - Trimms the Monitoring ID and Monitoring Token configuration
+         variables to ensure correctness.
 
-       Fixed the Xen Server 6 package repo
+       - Fixes the Xen Server 6 package repo.
```

Comment:
Issue:
The guideline doesn't seem to make sense and both the existing and updated examples don't seem to match that guideline.

> Provide an initial phrase that describes the issue that was fixed. Start the phrase with a past-tense verb

Suggestion:

I think this might need to be reworked to what you intended. Personally, I think "Resolved issues" should contain both what the behavior is now and what it used to be. E.g. 
"The driver now allows SRV record addresses that contain a trailing dot. Previous versions treated this as an invalid address."

Applying this to the first example, this could be:
"Supports Debian 8 (Jessie)" => "Now supports Debian 8 (Jessie). Prior versions were incompatible due to ..."

Reply:
@ccho-mongodb, good catch, I'll change the guidance to "present-tense." "Now" seems redundant in a release notes context. For Atlas, we usually start with a present tense verb.

Reply:
For the Debian example, I don't think it is necessary to do more explaining about why . For the 'Fixes the server' example, it would be nice to go into what was fixed to demonstrate that slightly more explanation is necessary.